README.md

```diff
@@ -7,6 +7,8 @@ Music Blocks is a programming language and collection of manipulative
 tools for exploring musical and mathematical concepts in an
 integrative and fun way.
 
+**Refer the following sections below to get familiar with this awesome project:**
```

Comment:
I don't think we need this line. And we don't need to toot our own horn with qualifiers like awesome.

Reply:
I will take back this change